### PR TITLE
Update instructions for initial setup

### PR DESCRIPTION
**Pull request by kant2002**
I do not expert in OPAM, but seems to be setting up currently can be easier with newer versions.
It seems to be by simply running `opam install .` you do not need even run `make` afterwards.

### CODE REVIEW

**Comment by nikswamy** [comment]
@tahina-pro , does this look correct to you?

**Comment by aseemr** [comment]
@tahina-pro, could you please take a look at it? @kant2002, do you mind resolving conflicts in the PR?

**Comment by tahina-pro** [comment]
Thanks a lot Andrii for your suggestions. I can confirm that these instructions work well in a [`ocaml/opam:ubuntu-ocaml-4.14` Docker image](https://hub.docker.com/r/ocaml/opam/tags?page=1&name=ubuntu-ocaml-4.14).
So, other than my small language comments, we are good to go!

**Comment by kant2002** [comment]
@tahina-pro I 100% sure about my issues with language. But seems to be you miss to post language comments itself

File: INSTALL.md [hunk 1]
```diff
@@ -245,8 +245,8 @@ Then follow step 4 in [Instructions for all OSes](#instructions-for-all-oses) be
      type `opam switch list-available`
      to see what versions are available and then `opam switch <version-number>`.
 
-   - Afterwards you can also install the `depext` package,
-     to be able to install some binary dependencies below more easily.
+   - Afterwards you can also install the `depext` package if you are on OPAM version lower then 2.1,
+     to be able to install some binary dependencies below more easily. Version of OPAM after 2.1 has depext handling backed in.
```
Comment (nikswamy) [inline INSTALL.md:249]: s/backed/baked/

Comment (kant2002) [reply INSTALL.md]: Done. Can I ask you try `suggestions` next time? I think that feature nice exact for this kind of issues which crops up.

**Comment by aseemr** [comment]
Thanks @kant2002!

**Pull request merged**